packages/terra-dynamic-grid/src/DynamicGrid.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { StyleSheet, css } from 'aphrodite';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import generateStyles from './styles';
+import Region from './Region';
+
+const propTypes = {
+  /**
+  * The child Region components that make up the grid.
+  */
+  children: PropTypes.node,
+  /**
+  * The layout configuration that defines the grid. Multiple layouts can be passed per mediaquery to make the grid responsive.
+  */
+  layouts: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
The prop-types doc seems to be blowing up on this.

<img width="869" alt="screen shot 2017-08-14 at 10 20 47 am" src="https://user-images.githubusercontent.com/633148/29278679-160f6ef4-80db-11e7-8071-ef424ac7589a.png">

I thought we accounted for `arrayOf`, but maybe that logic only made it into the markdown scripts. We may need to port that code over to the code that generates the react table.